pkgs/by-name/sq/sqlline/package.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  maven,
+  fetchFromGitHub,
+  makeWrapper,
+  jre,
+  docbook_xml_dtd_42,
+}:
+
+maven.buildMavenPackage rec {
+  pname = "sqlline";
+  version = "1.12.0";
+
+  src = fetchFromGitHub {
+    owner = "julianhyde";
+    repo = "sqlline";
+    rev = "sqlline-${version}";
```

See https://github.com/NixOS/nixpkgs/pull/355973 (you might need to rebase to latest master) 
```suggestion
    tag = "sqlline-${version}";
```